plugins/modules/remote_management/redfish/ilo_redfish_command.py

```diff
@@ -0,0 +1,343 @@
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      service_attributes:
+        ConnectSequenceTrace: "Disabled"
+        CustomDebugBlkIo: "Disabled"
+
+  - name: Verify bios attributes
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyBiosAttributes
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "*****"
+      bios_attributes:
+        SubNumaClustering: "Disabled"
+        WorkloadProfile: "Virtualization-MaxPerformance"
+
+  - name: Verify logical drives
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details: [{"LogicalDriveName": "LD1",
+                       "Raid": "Raid1",
+                       "DataDrives": {
+                          "DataDriveCount": 2,
+                          "DataDriveMediaType": "HDD",
+                          "DataDriveInterfaceType": "SAS",
+                          "DataDriveMinimumSizeGiB": 0
+                          }
+                      }]
+
+  - name: Verify UEFI boot order
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyUefiBootOrder
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      uefi_boot_order: ["Generic.USB.1.1"]
+
+  - name: Verify specified logical drives
+    ilo_redfish_command:
+      category: Systems
+      command: VerifySpecifiedLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details: [{"LogicalDriveName": "LD1",
+                       "Raid": "Raid1",
+                       "DataDrives": {
+                          "DataDriveCount": 2,
+                          "DataDriveMediaType": "HDD",
+                          "DataDriveInterfaceType": "SAS",
+                          "DataDriveMinimumSizeGiB": 0
+                          }
+                      }]
+
+  - name: Check server reboot status
+    ilo_redfish_command:
+      category: Systems
+      command: CheckRebootStatus
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get specified logical drives details
+    ilo_redfish_command:
+      category: Systems
+      command: CheckRebootStatus
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      logical_drives_names: ["LD1", "LD2"]
+
+'''
+
+RETURN = '''
+ilo_redfish_command:
+    description: Returns the status of the operation performed on the iLO.
+    type: dict
+    contains:
+        command:
+            description: Returns the output msg and whether the function executed successfully.
+            type: dict
+            contains:
+                ret:
+                    description: Return True/False based on whether the operation was performed succesfully.
+                    type: bool
```

Comment:
Instead of returning this non-standard denominated value, this should be the result of the `failed` state in the module output.

```python
if any(results[c]['ret'] is False for c in results):
    self.module.fail_json(**results)   # or something similar
```